data/map planets.txt

```diff
@@ -157,31 +157,31 @@ planet Ruxanguv
 	landscape land/desert4
-	description "Though too hot to be habitable by most species, this planet nonetheless harbours a fairly diverse biosphere comprised entirely of temperature-tolerant bacteria. Most Naltok biotechnology can trace at least part of its design to the extremely thermostable proteins produced by the bacteria here."
-	spaceport "The spaceport here is quite small, with docking tubes that connect your ship directly to the largest of the research centres on this planet. Despite the hostile surface conditions, this planet is home to several such locations, as the work of identifying and sequencing every bacterial species here is still ongoing."
+	description `Though too hot to be habitable by most species, this planet nonetheless harbors a fairly diverse biosphere comprised entirely of temperature-tolerant bacteria. Most Naltok biotechnology can trace at least part of its design to the extremely thermostable proteins produced by the bacteria here.`
+	spaceport `The spaceport here is quite small, with docking tubes that connect your ship directly to the largest of the research centers on this planet. Despite the hostile surface conditions, this planet is home to several such locations, as the work of identifying and sequencing every bacterial species here is still ongoing.`
 	"required reputation" 25
 
 planet Thosekrol
 	attributes naltok
 	landscape land/fields6
-	description "Insofar as the Telashki have a capital, this is it. In addition to its population of nearly ten billion, Thosekrol is also home to the headquarters of the Ministry of War, Ministry of Transport and Ministry of Minerals and to their second largest shipyards after Ngakego."
-	description "Rather than expanding outwards into the countryside, the cities here have been made taller and denser, leaving most of the planet untouched."
-	spaceport "The spaceport on Thosekrol is situated at the south pole, connected to the rest of the planet by an incredibly complex underground network of railways and canals. Even within the spaceport itself, there are a series of tiny passenger scuttles that efficiently transport visitors between the various shops and utilities."
+	description `Insofar as the Telashki have a capital, this is it. In addition to its population of nearly ten billion, Thosekrol is also home to the headquarters of the Ministry of War, the Ministry of Transport, and the Ministry of Minerals, with the addition of their second largest shipyards after Ngakego.`
+	description `	Rather than expanding outwards into the countryside, the cities here have been made taller and denser, leaving most of the planet untouched.`
+	spaceport `The spaceport on Thosekrol is situated at the south pole, connected to the rest of the planet by an incredibly complex underground network of railways and canals. Even within the spaceport itself, there are a series of tiny passenger shuttles that efficiently transport visitors between the various shops and utilities.`
```

Comment:
I did mean scuttles (referring to a type of light rail with small vehicles, generally running short connecting routes) but it seems that term is far less widely used than I thought, even within England, and even less so in the US. I'll probably change this to say trams instead, that probably fits better.

Reply:
```suggestion
	spaceport `The spaceport on Thosekrol is situated at the south pole, connected to the rest of the planet by an incredibly complex underground network of railways and canals. Even within the spaceport itself, there are a series of tiny trams that efficiently transport visitors between the various shops and utilities.`
```